packages/arb-token-bridge-ui/src/hooks/useBalance.ts

```diff
@@ -39,6 +39,52 @@ const merge: Middleware = (useSWRNext: SWRHook) => {
   }
 }
 
+const fetchErc20 = async ({
```

Comment:
Smol refactor: Moved `fetchErc20` code outside of the `useBalance` hook as it can be it's own pure function.